plot.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python2.7
+
+import matplotlib
+matplotlib.use('Agg')
+matplotlib.rcParams['text.usetex'] = True
+matplotlib.rcParams['text.latex.preamble'] = [r'\usepackage[cm]{sfmath}']
+matplotlib.rcParams['font.family'] = 'sans-serif'
+matplotlib.rcParams['font.sans-serif'] = 'cm'
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.polynomial.polynomial import polyfit
+import os
+
+sort = []
+
+with open("results.csv", "r") as f:
+    for l in f.readlines():
+        name, size, results = l.split(" ")
+        results2 = []
```

Comment:
`results2` but no `results`?

Reply:
Line 18 has `results`

Reply:
Oops, I'm an idiot!

BTW, you could simplify the loop to something like:
```python
results2 = [float(r) for r in results.split(",")]
```